.github/workflows/release.yml

```diff
@@ -43,17 +43,15 @@ jobs:
       - name: Release
         uses: cycjimmy/semantic-release-action@v3
         with:
-          semantic_version: 19
           extra_plugins: |
-            @semantic-release/commit-analyzer@^9
+            @semantic-release/commit-analyzer
```

Comment:
Since we're specifying Node 14 for workarounds, we need to specify versions for all these plugins in order for the release workflow to work correctly. Please copy the versions from https://github.com/googlemaps/google-maps-ios-utils/blob/main/.github/workflows/release.yml#L76-L82 which have been verified to work.

Reply:
@usefulthink see change requested above